src/golang/lib/models/shared/integration_config.go

```diff
@@ -0,0 +1,65 @@
+package shared
+
+import (
+	"github.com/dropbox/godropbox/errors"
+)
+
+type S3ConfigType string
+
+const (
+	AccessKeyS3ConfigType         S3ConfigType = "access_key"
+	ConfigFilePathS3ConfigType    S3ConfigType = "config_file_path"
+	ConfigFileContentS3ConfigType S3ConfigType = "config_file_content"
+)
+
+// S3IntegrationConfig contains the fields for connecting an S3 integration.
```

Comment:
Renamed S3Config & GCSConfig to S3IntegrationConfig & GCSIntegrationConfig because it has the same name as the storage configs. LMK if you think I should rename the other integration configs to be [service]IntegrationConfig rather than just [service]Config

Reply:
I think these are the only configs with named configs right? For example we don't have a `PostgresConfig` or `MySQLConfig`, etc.

Reply:
There is an AirflowConfig too. The rest in the file are named: S3IntegrationConfig, GCSIntegrationConfig, K8sIntegrationConfig, LambdaIntegrationConfig. Updated AirflowConfig to be AirflowIntegrationConfig